src/modules/Lan.rb

```diff
@@ -788,7 +788,6 @@ def Export
         "config"               => NetworkConfig.Export,
         "devices"              => devices,
         "ipv6"                 => @ipv6,
-        "routing"              => profile.routing,
```

Comment:
hmm, I am not sure if we should skill it completely or just use empty hash instead.